include/bluetooth/audio/mics.h

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_MICS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_MICS_H_
+
+/**
+ * @brief Microphone Input Control Service (Microphone Input Control Service)
```

Comment:
Probably a Doxygen thing I do not know, but why is the brief text repeated?

Reply:
One is the brief, the other is the name of the group :) They just happen to often have the same value